### PR TITLE
Change default log level of network costs to info

### LINKED ISSUE #3263: [Feature] Add configuration to change LOG_LEVEL on network-costs
### Problem Statement

Change #3206 Added LOG_LEVEL to `cost-analyzer/templates/cost-analyzer-network-costs-template.yaml`.  This change has caused a large amount of logs to be generated every 5 seconds.

### Solution Description

The LOG_LEVEL value needs to be configurable via the helm chart so consumers can reduce or increase the logs as needed.

### Alternatives

_No response_

### Additional Context

Example of logs

```
2024-03-22T10:56:01-05:00  INFO network_costs_rs::network::watcher: Processed 1974 Connection Entries
2024-03-22T10:56:06-05:00  INFO network_costs_rs::network::watcher: Processed 1960 Connection Entries
2024-03-22T10:56:11-05:00  INFO network_costs_rs::network::watcher: Processed 1960 Connection Entries
2024-03-22T10:56:16-05:00  INFO network_costs_rs::network::watcher: Processed 1906 Connection Entries
2024-03-22T10:56:21-05:00  INFO network_costs_rs::network::watcher: Processed 1888 Connection Entries
2024-03-22T10:56:26-05:00  INFO network_costs_rs::network::watcher: Processed 1870 Connection Entries
2024-03-22T10:56:31-05:00  INFO network_costs_rs::network::watcher: Processed 1832 Connection Entries
2024-03-22T10:56:36-05:00  INFO network_costs_rs::network::watcher: Processed 1932 Connection Entries
2024-03-22T10:56:41-05:00  INFO network_costs_rs::network::watcher: Processed 1962 Connection Entries
2024-03-22T10:56:46-05:00  INFO network_costs_rs::network::watcher: Processed 2016 Connection Entries
2024-03-22T10:56:51-05:00  INFO network_costs_rs::network::watcher: Processed 2026 Connection Entries
2024-03-22T10:56:56-05:00  INFO network_costs_rs::network::watcher: Processed 2024 Connection Entries
2024-03-22T10:57:01-05:00  INFO network_costs_rs::network::watcher: Processed 2072 Connection Entries
```

related issue: https://github.com/kubecost/cost-analyzer-helm-chart/issues/2309

FYI: @cliffcolvin 

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a feature request only for the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR change?
- Makes the log level configurable for the network costs daemonset

## Does this PR rely on any other PRs?
- no

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
- It allows users the ability to enable different log types for the network costs pods `error, warn, info, debug`
## Links to Issues or tickets this PR addresses or fixes
Closes #3263

## What risks are associated with merging this PR? What is required to fully test this PR?
- There is little risk, this is an enhancement to the env variable, which allows different log levels to be defined
- To test this change, you mainly just need to package helm locally, and supply different logs via `values.yaml` and see if they work.

## How was this PR tested?
- deployed a kind cluster
- packaged the helm chart locally
- supplied different log values and verified they were enabled. 

Screenshot 2024-03-27 at 9 35 09 AM

Screenshot 2024-03-27 at 9 34 21 AM

### CODE REVIEW

File: cost-analyzer/cost-analyzer-2.2.0.tgz
Comment (chipzoller) [inline cost-analyzer/cost-analyzer-2.2.0.tgz:1]: Don't need this.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: cost-analyzer/cost-analyzer-2.2.0.tgz
Comment (ghost) [reply cost-analyzer/cost-analyzer-2.2.0.tgz:1]: oops, that was the local chart i tested

**Review by chipzoller — APPROVED** [review]

**Comment by chipzoller** [comment]
When you send a PR which addresses an issue (in whole), please link the PR to the issue so it is closed automatically. You can do so manually or using GitHub's [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). I have made this change for you. Please also try and make the title as descriptive as possible. Usernames or other prefixes are not necessary.

**Pull request merged**